pcdsdevices/mirror.py

```diff
@@ -752,12 +752,16 @@ class XOffsetMirrorNoBend(XOffsetMirror):
 
     name : str
 
-    Currently (10/11/2023) services: mr1l1
+    Currently (10/11/2023) services: mr1l1, mr1k3, mr2k3
```

Comment:
Is the old date still correct in this comment?
```suggestion
    Currently (5/15/2024) services: mr1l1, mr1k3, mr2k3
```